src/CloudApi/CloudCredentials.php

```diff
@@ -58,12 +58,6 @@ public function getCloudKey(): ?string {
       return $this->datastoreCloud->get('acli_key');
     }
 
-    // Legacy format.
```

Comment:
@grasmash legacy cloud configs (with a "key" instead of "keys") throw an error. Did you intentionally deprecate that format?

If so, I'll remove all associated legacy code like this.

If not, please restore support and I'll add a test case (as below)